src/mmap_unix.rs

```diff
@@ -154,6 +154,8 @@ impl MmapRegion {
             return Err(Error::Mmap(io::Error::last_os_error()));
         }
 
+        let _ret = unsafe { libc::madvise(addr, size, libc::MADV_HUGEPAGE) };
```

Comment:
Hi! Apologies for the late reply :( I was just wondering, what's the difference between using this `madvise` call and providing `MAP_HUGETLB` as part of `flags`?

Reply:
MAP_HUGETLB uses hugetlbfs, MADV_HUGEPAGE uses transparent huge page for memfd or anonymous mapping.